docs/content/introduction/productvisie.md

```diff
@@ -3,21 +3,23 @@
 ## Inhoud
 * [Introductie](#introductie)
 * [Productvisie ZDS](#productvisie-zds)
-* [Centraal aanbieden](#centraal-aanbieden)
+* [Scope](#scope)
 * [Uitgangspunten](#uitgangspunten)
 * [Realisatie](#realisatie)
+* [Transitie](#transitie)
+* [Centraal aanbieden](#centraal-aanbieden)
 * [Gerelateerde trajecten](#gerelateerde-trajecten)
 
 
 ## Introductie
```

Comment:
De "toegevoegde waarde" die beschreven moet worden bestaat wellicht uit een beschrijving van "het waarom", voorafgaand aan dit stuk in de Introductie. Suggesties zeer welkom.

Reply:
Kunnen we hier wat mee?:

Elementen van de toegevoegde waarde voor gemeenten zijn:
- Kortere doorlooptijd van het inrichten van nieuwe koppelingen (plug and play is veelgehoorde wens)
- Lagere ontwikkelkosten van koppelingen op basis van ZDS
- Lagere beheerkosten door backwards compatibiliteit (wijzigingen in de standaard leiden meestal niet tot aanpassingen in bestaande koppelingen)
- Voorkomen lock-in door echt uitwisselbare componenten
- Hogere kwaliteit van de standaard doordat deze in de praktijk en met medewerking van gemeenten en leveranciers is ontwikkeld. En ook door parallel aan de standaard een werkende referentie-implementatie te ontwikkelen.

Toegevoegde waarde voor leveranciers:
- Lagere ontwikkelkosten van het koppelvlak
- Kunnen zich onderscheiden op onderdelen die burgers, bedrijven en medewerkers raken (i.p.v. te concurreren op infrastructuur)

Toegevoegde waarde voor VNG Realisatie:
- Beheer van de standaard is eenvoudiger doordat aanpassingen gemakkelijker zijn door te voeren 
- Kwaliteitscontrole van koppelvlakken van leveranciers is eenvoudiger en daarmee goedkoper.
- Backwards compatibiliteit is eenvoudiger te behouden

Reply:
Dank, toegevoegd en pushed naar deze PR